index.html

```diff
@@ -14,172 +14,200 @@
+                    contact me.
+                </p>
+                <a href="#">LET’S CONNECT</a>
+                <ul class="social-links">
+                    <li><a href="#"><img src="img/icons/Vector.png" alt=""></a></li>
+                    <li><a href="#"><img src="img/icons/Linkedin_icon.png" alt=""></a></li>
+                    <li><a href="#"><img src="img/icons/Vector-1.png" alt=""></a></li>
+                    <li> <a href="#"><img src="img/icons/Vector-2.png" alt=""></a></li>
+                    <li> <a href="#"><img src="img/icons/Vector3.png" alt=""></a></li>
+                </ul>
+            </div>
         </section>
         <section class="work-cards-container">
             <div class="work-card work-card1">
-                <img src="img/works/work1.png" alt="work1">
-
-                <h2 class="work-title">Tonik</h2>
-                <div class="work-info">
-                    <p class="first">CANOPY</p>
-                    <p class="second">• Back End Dev</p>
-                    <p class="second">• 2015</p>
-                </div>
-                <div class="word-desc">
-                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                <div class="card-img">
+                    <img src="img/works/work1.png" alt="work1">
                 </div>
-                <div class="work-lang">
-                    <ul>
-                        <li>
-                            <p class="lang-text">html</p>
-                        </li>
-                        <li>
-                            <p class="lang-text">css</p>
-                        </li>
-                        <li>
-                            <p class="lang-text">javascript</p>
-                        </li>
-                    </ul>
-                </div>
-                <button type="button" class="work-btn btn">
+                <div class="card-details">
+                    <h2 class="work-title">Tonik</h2>
+                    <div class="work-info">
+                        <p class="first">CANOPY</p>
+                        <p class="second">• Back End Dev</p>
+                        <p class="second">• 2015</p>
+                    </div>
+                    <div class="word-desc">
+                        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    </div>
+                    <div class="work-lang">
+                        <ul>
+                            <li>
+                                <p class="lang-text">html</p>
+                            </li>
+                            <li>
+                                <p class="lang-text">css</p>
+                            </li>
+                            <li>
+                                <p class="lang-text">javascript</p>
```

Comment:
- [ ] please try to add some padding to your tag in order to match the Figma design 
- Figma 
![image](https://user-images.githubusercontent.com/61790587/171683086-afce4712-292a-413b-91f4-6ccdb7eb5b03.png)
- Yours 
![image](https://user-images.githubusercontent.com/61790587/171683157-e0a579bb-d8ab-4ef9-8d13-25acf4af4e9a.png)